lib/components/dashboard/degree-info/distributionFunctions.tsx

```diff
@@ -101,7 +101,9 @@ const handleTagType = (
       break;
     case 'A': // Area
       updatedConcat = (
-        course.areas !== 'None' && course.areas.includes(splitArr[index])
+        course.areas !== undefined &&
+        course.areas !== 'None' &&
+        course.areas.includes(splitArr[index])
```

Comment:
am now thinking this should be area not areas. It's area in main/dev but areas in DIS branch, hence the confusion.

Reply:
nm